agent/exec/controller.go

```diff
@@ -3,6 +3,7 @@ package exec
 import (
 	"fmt"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
Any reason for `logrus` to be imported here when it seems that all logs are suppose to be using `swarmkit/log` ?